### PR TITLE
Merge upstream 3.5.3 

### PR DESCRIPTION
**Pull request by snovikov**
Update module from upstream https://github.com/terraform-aws-modules/terraform-aws-atlantis

### CODE REVIEW

File: .github/workflows/pre-commit.yml [hunk 1]
```diff
@@ -2,98 +2,77 @@ name: Pre-Commit
 
 on:
   pull_request:
-  push:
     branches:
+      - main
```
Comment (cytopia) [inline .github/workflows/pre-commit.yml:6]: We're not using `main`

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
+name: Release
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```
Comment (cytopia) [inline .github/workflows/release.yml:7]: No `main`

File: .releaserc.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
+{
+  "branches": [
+    "main",
```
Comment (cytopia) [inline .releaserc.json:3]: No `main`

**Review by cytopia — APPROVED** [review]
As discussed, leaving it in does no harm either

File: .github/workflows/pre-commit.yml (hunk 1 above)
Comment (snovikov) [reply .github/workflows/pre-commit.yml:6]: `master` is also there

**Pull request merged**